### ISSUE TITLE
The iteration number of train to stop?

### ISSUE THREAD

**Issue by zwhinmedia** [chuanqi305/MobileNet-SSD#66]
Hi chuanqi

I have used your code to my own datasets. After four days training, the training resut is that:
"
I0128 17:13:54.750319 32754 sgd_solver.cpp:138] Iteration 66980, lr = 0.000125
I0128 17:14:42.158315 32754 solver.cpp:243] Iteration 66990, loss = 1.16429
I0128 17:14:42.158717 32754 solver.cpp:259]     Train net output #0: mbox_loss = 1.14603 (* 1 = 1.14603 loss)
I0128 17:14:42.301295 32754 sgd_solver.cpp:138] Iteration 66990, lr = 0.000125
I0128 17:15:24.865245 32754 solver.cpp:596] Snapshotting to binary proto file snapshot/mobilenet_iter_67000.caffemodel
I0128 17:15:24.941534 32754 sgd_solver.cpp:307] Snapshotting solver state to binary proto file snapshot/mobilenet_iter_67000.solverstate
I0128 17:15:29.573871 32754 solver.cpp:243] Iteration 67000, loss = 1.10252
I0128 17:15:29.573935 32754 solver.cpp:259]     Train net output #0: mbox_loss = 1.05099 (* 1 = 1.05099 loss)
I0128 17:15:30.090251 32754 sgd_solver.cpp:138] Iteration 67000, lr = 0.000125
I0128 17:16:20.436995 32754 solver.cpp:243] Iteration 67010, loss = 1.06106
I0128 17:16:20.437379 32754 solver.cpp:259]     Train net output #0: mbox_loss = 0.96547 (* 1 = 0.96547 loss)
I0128 17:16:21.063503 32754 sgd_solver.cpp:138] Iteration 67010, lr = 0.000125”

And I have tested the result, results is that:
"
I0128 17:02:17.709306 31817 annotated_data_layer.cpp:62] output data size: 24,3,300,300
I0128 17:02:19.036150 31817 parallel.cpp:425] Starting Optimization
I0128 17:02:19.036249 31817 solver.cpp:294] Solving MobileNet-SSD
I0128 17:02:19.036262 31817 solver.cpp:295] Learning Rate Policy: multistep
I0128 17:02:19.036293 31817 blocking_queue.cpp:50] Data layer prefetch queue empty
I0128 17:02:21.871599 31817 solver.cpp:332] Iteration 0, loss = 0.901741
I0128 17:02:21.871665 31817 solver.cpp:433] Iteration 0, Testing net (#0)
I0128 17:02:21.885555 31817 net.cpp:693] Ignoring source layer mbox_loss
I0128 17:05:50.124867 31817 solver.cpp:546]     Test net output #0: detection_eval = 0.967494
I0128 17:05:50.125097 31817 solver.cpp:337] Optimization Done.
I0128 17:05:50.288264 31817 caffe.cpp:254] Optimization Done.
"

So I want to know if the result is good and how to stop the training? wait training to stop itself?
